v2.0/src/Components/Pages/ActionsPage/ActionsPage.js

```diff
@@ -265,14 +271,21 @@ class ActionsPage extends React.Component {
   };
   // renders all the actions
   renderActions(actions) {
-    if (!actions || actions.length === 0) {
+    if (!actions) {
       return (
         <p>
           There aren't any actions available in this community yet, come back
           later.
         </p>
       );
     }
+    if (actions.length === 0) {
```

Comment:
nice! here we can use actions.length because we already verified at the top that `actions` is not empty.